python/pipeline/preprocess.py

```diff
@@ -185,8 +190,11 @@ class ExtractRaw(dj.Imported):
 
     @property
     def key_source(self):
-        return Prepare() * Method().proj() & dj.OrList(
-            Prepare.Aod() * Method.Aod(), Prepare.Galvo() * Method.Galvo())
+        return Prepare() * Method() \
+               & ((Prepare.Galvo() * Method.Galvo() - 'segmentation="manual"') | \
```

Comment:
the __or__ operator is not implemented.  What is meant here?  Is this copied from MATLAB?  Use `dj.OrList` instead.